templates/cd/cd-header/cd-ocha.html.twig

```diff
@@ -1,10 +1,13 @@
 
 
-<div class="cd-ocha">
+<section id="cd-ocha-services" class="cd-ocha">
 
   {# Dropdown toggle is created with javascript #}
 
-  <div class="cd-global-header__dropdown cd-ocha-dropdown cd-dropdown" id="cd-ocha-dropdown" aria-label="{{ 'OCHA Services'|t }}" data-cd-toggable="{{ 'OCHA Services'|t }}" data-cd-hidden="true" data-cd-icon="arrow-down" data-cd-logo="ocha-logo" data-cd-component="cd-ocha">
+  <div class="cd-global-header__dropdown cd-ocha-dropdown cd-dropdown" id="cd-ocha-dropdown" aria-labelledby="cd-ocha-services-title" data-cd-toggable="OCHA Services" data-cd-hidden="true" data-cd-icon="arrow-down" data-cd-logo="ocha-logo" data-cd-component="cd-ocha">
```

Comment:
@left23 Possibly out of scope of this PR but the OCHA services section would be a good candidate to use the `data-cd-replace` attribute, moving the `h2` above this `div`.

Reply:
@orakili Yes, I agree. Adding it.